assignments/prototypes.js

```diff
@@ -16,13 +16,42 @@
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
 
+
+function GameObject (goAttributes) {
+  this.createdAt = new Date ();
+  this.name = goAttributes.name;
+  this.dimensions = goAttributes.dimensions;
+}
+
+GameObject.prototype.destroy = function () {
+  return `${this.name} was removed from the game.` ;
+}
+
+
+
+
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
 
+
+function CharacterStats (stats) {
+  GameObject.call (this, stats);
+  this.healthPoints = stats.healthPoints;
+  
```

Comment:
Great job binding the this keyword to the parent object. I know it can be tricky to remember to call the binding method on the parent and pass it the childs attributes as an argument.